packages/git-semver-tags/index.js

```diff
@@ -13,7 +13,11 @@ function lernaTag (tag, pkg) {
   }
 }
 
-module.exports = function (callback, opts) {
+module.exports = function (opts, callback) {
```

Comment:
I think you would need to modify all the callers of this method among the sibling packages, as well? Probably the tests, too.

Reply:
Oooh. Yea, sorry.

Reply:
Actually, tests are good. Will add one more for when there is `opts`.